front/src/modules/ui/layout/show-page/components/ShowPageRightContainer.tsx

```diff
@@ -32,10 +32,10 @@ const StyledTabListContainer = styled.div`
   border-bottom: ${({ theme }) => `1px solid ${theme.border.color.light}`};
   box-sizing: border-box;
   display: flex;
-  gap: ${({ theme }) => theme.spacing(1)};
+  gap: ${({ theme }) => theme.spacing(2)};
   height: 40px;
-  padding-left: ${({ theme }) => theme.spacing(1)};
-  padding-right: ${({ theme }) => theme.spacing(1)};
+  padding-left: ${({ theme }) => theme.spacing(0)};
```

Comment:
I think you can go with `padding: 0` or remove the padding if there's no inherited padding maybe ?